contracts/Relay.sol

```diff
@@ -316,4 +325,14 @@ contract Relay is Ownable, Pausable, Initializable {
 
         return ok;
     }
+  
+    function checkDuplicates(address[] memory list) internal pure returns (bool){
+        for (uint index = 0; index < list.length; index++) {
+            for (uint k = index + 1; k < list.length; k++) {
+                if(list[index] == list[k]) {
+                    revert('Relay:: checkDuplicates: Duplicate entries in list');
```

Comment:
This is actually a util function, suggest to return false, and `require()` in the upstream call function.

https://github.com/cryptofinlabs/cryptofin-solidity/blob/dafccd0a0aa184be4270db226bc5ef99d7376a24/contracts/array-utils/AddressArrayUtils.sol#L299